pkg/config/config.go

```diff
@@ -1,6 +1,8 @@
 /*
 Copyright 2018 Heptio Inc.
 
+© 2022 Nokia
```

Comment:
Same comment as other copyright.

Reply:
The copyright message has been updated as suggested, in all relevant files (I left out the `.golden` files)